webcodecs/audioDecoder-codec-specific.https.any.js

```diff
@@ -368,7 +371,30 @@ promise_test(async t => {
 
   await decoder.flush();
   assert_equals(outputs, CONFIG.codec === 'vorbis' ? 1 : 2, 'outputs');
-}, 'Test decoding a with negative timestamp');
+}, 'Test decoding a with a negative timestamp');
+
+promise_test(async t => {
+  const callbacks = {};
+  const decoder = createAudioDecoder(t, callbacks);
+
+  let outputs = 0;
+  callbacks.output = frame => {
+    if (outputs === 0) {
+      assert_equals(frame.timestamp, 42);
+    }
+    outputs++;
+    frame.close();
+  };
+
+  decoder.configure(CONFIG);
+  decoder.decode(new EncodedAudioChunk(
+      {type: 'key', timestamp: 42, data: CHUNK_DATA[0]}));
+  decoder.decode(new EncodedAudioChunk(
+      {type: 'key', timestamp: CHUNKS[0].duration + 42, data: CHUNK_DATA[1]}));
+
+  await decoder.flush();
+  assert_equals(outputs, CONFIG.codec === 'vorbis' ? 1 : 2, 'outputs');
+}, 'Test decoding a with a positive timestamp');
```

Comment:
It looks like 'Test decoding' test is already doing it.

Reply:
Many thanks for taking a look. Maybe I'm missing something but as far as I can tell the 'Test decoding' test starts with a timestamp of exactly zero. This test is meant to check if the positive timestamp is carried over to the first `AudioData` instance.

Reply:
got it, thanks